### DISCUSSION TITLE
Internalisation (multilingual) mode after upgrade to v1

### DISCUSSION THREAD

**Discussion by sarviun** [thuliteio/doks#1211] [Q&A]
I've just moved to v1.4.0 and it seems the translations by content directory don't work anymore [(Hugo docs)](https://gohugo.io/content-management/multilingual/#translation-by-content-directory). Previous version has been set up with language hierarchy and ready to load with files though. On the other hand, [translation by name](https://gohugo.io/content-management/multilingual/#translation-by-file-name) works.

**Comment by sarviun** [comment]
Ok, after some testing I found out I have to force the language code in URL although I set up the `defaultContentLanguage = "en"`. If I have e.g. folder content/en/contact/index.md, [localhost:port/contact]() returns 404, but [localhost:port/en/contact]() returns the page. Maybe this is intended behaviour but it has worked in the previous version. How do I make it redirect to the default language content?

**Comment by h-enk** [comment]
Did you set `defaultContentLanguageInSubdir ` in `hugo.toml`?

```toml
defaultContentLanguage = 'en'
defaultContentLanguageInSubdir = false
```

See also [Multilingual example](https://gohugo.io/content-management/urls/#permalinks-multilingual-example)
